CS342MemberViews/Sources/BrianPark/BrianPark.swift

```diff
@@ -0,0 +1,56 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import InstructorViews
+import SwiftUI
+import MapKit
+
+
+public struct BrianPark: View {
+    @Binding var navigationPath: NavigationPath
+    
+    
+    public var body: some View {
+        ScrollView(.vertical) {
+            VStack(spacing: 42) {
+                ProfileView()
+//                ProfileView(profileImage: Bundle.module.image(fromFileNamed: "BrianPark"))
+                DescriptionView()
+                Text("Most recently from:")
+                    .font(.title3)
+                    .padding()
+                MapView()
+                    .frame(height: 300)
+                    .offset(y:-50)
```

Comment:
Swiftlint is throwing an error here because the colon should have a single space after it.

```suggestion
                    .offset(y: -50)
```